modules/currency-l0/src/main/scala/org/tessellation/currency/http/routes/SnapshotRoutes.scala

```diff
@@ -0,0 +1,67 @@
+package org.tessellation.currency.http.routes
+
+import cats.effect.Async
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+
+import org.tessellation.currency.schema.currency.CurrencySnapshot
+import org.tessellation.ext.codecs.BinaryCodec
+import org.tessellation.ext.http4s.headers.negotiation.resolveEncoder
+import org.tessellation.kryo.KryoSerializer
+import org.tessellation.sdk.domain.snapshot.storage.SnapshotStorage
+import org.tessellation.sdk.ext.http4s.SnapshotOrdinalVar
+import org.tessellation.security.signature.Signed
+
+import io.circe.Encoder
+import io.circe.shapes._
+import org.http4s.circe.CirceEntityEncoder
+import org.http4s.dsl.Http4sDsl
+import org.http4s.server.Router
+import org.http4s.{EntityEncoder, HttpRoutes}
+import shapeless.HNil
+import shapeless.syntax.singleton._
+
+final case class SnapshotRoutes[F[_]: Async: KryoSerializer](
```

Comment:
It will also be extracted in next changes and reused (by passing prefix as param, for Global it will be `global-snapshot` and for Currency it will be `snapshot`).